src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1716,7 +1716,7 @@ void window_guest_rides_paint(rct_window* w, rct_drawpixelinfo* dpi)
         auto ride = get_ride(peep->FavouriteRide);
         if (ride != nullptr)
         {
-            ride->FormatNameTo(gCommonFormatArgs);
+            ride->FormatNameTo(ft);
```

Comment:
Not resetting `formatter` here meant that all guests were showing "N/A". See PR mentioned above